exporter/stats/stackdriver/stackdriver.go

```diff
@@ -101,9 +101,25 @@ type Options struct {
 	Resource *monitoredrespb.MonitoredResource
 }
 
+// Enforces the singleton on NewExporter per process
+// lest there will be races with Stackdriver.
+// See Issue https://github.com/census-instrumentation/opencensus-go/issues/258
+var newExporterOnce sync.Once
+var errSingletonExporter = errors.New("only one exporter can be created per process")
+
 // NewExporter returns an exporter that uploads stats data to Stackdriver Monitoring.
-// Only one Stackdriver exporter should be created per process.
+// Only one Stackdriver exporter should be created per process, any subsequent
```

Comment:
We should allow multiple exporters if the project ID is different. We should only return an error if the user is trying to initiate a new exporter for the same project ID.

Reply:
Oh okay, thanks for the clarification.